### DISCUSSION TITLE
Disable CleanupUnusedClaimTask

### DISCUSSION THREAD

**Discussion by LucidAPs** [GriefPrevention/GriefPrevention#1314] [General]
How can I completely disable this task?
![image](https://user-images.githubusercontent.com/42933023/111077926-bb5ca200-84fb-11eb-8ca5-86313efb9bd2.png)

**Comment by Jikoo** [comment]
You can't. What you can do is set `GriefPrevention.Advanced.ClaimExpirationCheckRate` to a high value so claim expiration runs very infrequently - the task will run once after the server has been up for a minute but then your long delay will kick in and it will effectively never run again.
